tests/ftests/081-pybindings-cgrp_get_curr_ctrl_path-v1.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_get_current_controller_path() test using the python bindings (cgroup v1)
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli, Mode
+from libcgroup import Cgroup, Version
+from process import Process
+import consts
+import ftests
+import sys
+import os
+
+
+CGNAME = '081getctrlpathv1'
+CONTROLLER = 'cpu'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if Cgroup.cgroup_mode() == Mode.CGROUP_MODE_UNIFIED:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the legacy cgroup v1 hierarchy'
+
+    return result, cause
+
+
+def setup(config):
+    CgroupCli.create(config, CONTROLLER, CGNAME)
+
+    config.process.create_process_in_cgroup(config, CONTROLLER, CGNAME, ignore_systemd=True)
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    expected_path = os.path.join("/" + CGNAME)
+    pid = CgroupCli.get_pids_in_cgroup(config, CGNAME, CONTROLLER)[0]
+
+    cgrp = Cgroup(CGNAME, Version.CGROUP_V2)
+    #
+    # Test 1 - get the relative path of cgroup, for the pid's cpu controller.
+    #          It's expected to pass because we had created cgroup on cpu
+    #          hierarchy and moved the task to that group.
+    #
+    cgrp_path = cgrp.get_current_controller_path(pid, CONTROLLER)
+    if cgrp_path != expected_path:
+        result = consts.TEST_FAILED
+        cause = 'Expected cgroup path {} got {}'.format(expected_path, cgrp_path)
+
+    #
+    # Test 2 - get the relative path of cgroup, for the pid's memory controller.
+    #          It's expected to fail because we not had created cgroup.
+    #
+    cgrp_path = cgrp.get_current_controller_path(pid, "memory")
+    if cgrp_path == expected_path:
+        result = consts.TEST_FAILED
+        tmp_cause = 'cgroup path unexpectedly formed {}'.format(cgrp_path)
+        cause = '\n'.join(filter(None, [cause, tmp_cause]))
+
+    #
+    # Test 3 - get the relative path of cgroup, for the pid's invalid controller.
+    #          It's expected to fail because such controller doesn't exists.
+    #
+    try:
+        cgrp_path = cgrp.get_current_controller_path(pid, "invalid")
+    except RuntimeError as re:
+        if '50011' not in str(re):
```

Comment:
RuntimeError has access to the return code.  You should be able to do this:
```
if re.ret != 50011:
    raise re
```

Reply:
`RunError` has access to `ret`, that exception is raised when we run commands, this via pybindings and trying to check for `re.ret` will error out  with `'RuntimeError' object has no attribute 'ret'`

Reply:
D'oh.  You're totally right.  I obviously confused the two :(